client/src/pages/Review/Review.tsx

```diff
@@ -36,19 +56,9 @@ const Review = () => {
     onMainMarginLeft();
     window.addEventListener("resize", onMainMarginLeft);
     return () => window.removeEventListener("resize", onMainMarginLeft);
-  }, [navRef.current, mainRef.current, currentCourseKey]);
+  }, [navRef.current, mainRef.current]);
 
-  // At first render of the page, check if the course key is valid
-  // and assign value to current course key
   useEffect(() => {
-    if (
-      courseKeyUrlParam === undefined ||
-      courseKeyLookup[courseKeyUrlParam] === undefined
-    ) {
-      setCurrentCourseKey(null);
-    } else {
-      setCurrentCourseKey(courseKeyUrlParam);
-    }
```

Comment:
Relates to previous _url param_ comment. This guards only valid `course_id` in _url params_ is parsed and redirected.